source/javascripts/govuk/modules.js

```diff
@@ -0,0 +1,57 @@
+(function($, root) {
```

Comment:
Assuming this file has come from _somewhere else_, so not looking to closely at it.